CSS/style.css

```diff
@@ -1,195 +1,332 @@
+  line-height: 32px;
+  color: #18191f;
 }
 
 .feature-list-content {
-    font-size: 16px;
-    line-height: 26px;
-    color:#18191F;
+  font-size: 16px;
+  line-height: 26px;
+  color: #18191f;
 }
 
 .second-row {
-    display: flex;
-    align-items: center;
-    gap: 30px;
-    margin: 64px 165px auto;
+  display: flex;
+  align-items: center;
+  gap: 30px;
+  margin: 64px 165px auto;
+}
+
+.third-section-container {
+  background-color: #d5fafc;
+}
+
+.third-section {
+  max-width: 1440px;
+  min-height: 896px;
+  position: relative;
+  margin: auto;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+}
+
 
+.third-section-heading::before {
+  content: "";
+  position: absolute;
+  background-image: url("/Images/Third-Section/quotations-large.png") ;
+  background-size: contain;
+  background-repeat: no-repeat;
+  top: -50px;
+  left: -50px;
+  width: 150px;
+  height: 150px;
+  z-index: -1;
 
 }
 
+.third-section-heading {
+  gap: 8px;
+  width: 445px;
+  height: 168px;
+  z-index: 2;
+  position: relative;
+  margin-top: 100px;
+}
+
+.third-section-title {
+  font-weight: 800;
+  font-size: 48px;
+  line-height: 64px;
+  color: #18191f;
+}
+
+.third-section-description {
+  font-size: 18px;
+  line-height: 32px;
+}
+
+.testimonial-logos {
+  margin: 32px auto auto 32px;
+}
+
+.testimonial1 {
+  width: 350px;
+  height: 468px;
+  background: #ffffff;
+  box-shadow: 0px 10px 20px rgba(41, 41, 42, 0.07);
+  border-radius: 8px;
+  position: relative;
+  margin: 24px auto 116px 95px;
+
+}
+
+.testimonial1-description {
+  font-size: 18px;
+  line-height: 32px;
+  width: 254px;
+  height: 256px;
+  margin: 32px 32px auto 64px;
+}
+
+.testimonial2 {
+  width: 445px;
+  height: 340px;
+  background-color: #ffffff;
+  box-shadow: 0px 10px 20px rgba(41, 41, 42, 0.07);
+  border-radius: 8px;
+  position: relative;
+  margin: 90px auto auto 30px;
+}
+
+.testimonial2-description {
+  font-size: 18px;
+  line-height: 32px;
+  width: 349px;
+  height: 128px;
+  margin: 32px 32px auto 64px;
+}
+
+.testimonial3 {
+  width: 350px;
+  height: 276px;
+  background-color: #ffffff;
+  box-shadow: 0px 10px 20px rgba(41, 41, 42, 0.07);
+  border-radius: 8px;
+  position: relative;
+  margin: 30px auto auto 30px;
+}
+
+.testimonial3-description {
+  font-size: 18px;
+  line-height: 32px;
+  width: 254px;
+  height: 64px;
+  margin: 32px 32px auto 64px;
+  
+}
+
+.testimonial-person-name {
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 28px;
+  margin: 24px auto auto 64px;
+}
+
+.testimonial-person-position {
+  font-weight: 500;
+  font-size: 14px;
+  line-height: 24px;
+  height: 24px;
+  color: #969bab;
+  margin: auto auto 32px 64px;
+}
+
+.quotations-small {
+  position: absolute;
+  top: 104px;
+  left: 32px;
+}
+
+@media screen and (max-width: 900px) {
+  .third-section{
+    flex-direction: column;
+    padding: 100px 0;
+  }
+  .testimonial1 {
+    width: 445px;
+    height: 468px;
+    background-color: #ffffff;
```

Comment:
- [ ] If you didn't change the background color after `900px` you can delete this extra line
- [ ] Same for `border-radius` and `box-shadow` . If you  didn't change them after 900px you should delete these lines